src/ReCaptchaResponse.php

```diff
@@ -1,56 +1,46 @@
-<?php
+<?php declare(strict_types = 1);
 
 namespace Contributte\ReCaptcha;
 
-/**
- * @author Milan Felix Sulc <sulcmil@gmail.com>
- */
 final class ReCaptchaResponse
 {
 
 	// Error code list
-	const ERROR_CODE_MISSING_INPUT_SECRET = 'missing-input-secret';
-	const ERROR_CODE_INVALID_INPUT_SECRET = 'invalid-input-secret';
-	const ERROR_CODE_MISSING_INPUT_RESPONSE = 'missing-input-response';
-	const ERROR_CODE_INVALID_INPUT_RESPONSE = 'invalid-input-response';
-	const ERROR_CODE_UNKNOWN = 'unknow';
+	public const ERROR_CODE_MISSING_INPUT_SECRET = 'missing-input-secret';
+	public const ERROR_CODE_INVALID_INPUT_SECRET = 'invalid-input-secret';
+	public const ERROR_CODE_MISSING_INPUT_RESPONSE = 'missing-input-response';
+	public const ERROR_CODE_INVALID_INPUT_RESPONSE = 'invalid-input-response';
+	public const ERROR_CODE_UNKNOWN = 'unknow';
 
 	/** @var bool */
 	private $success;
 
-	/** @var string */
+	/** @var string[]|string|null */
```

Comment:
It could be probably only `string[]`, but I am not sure about data given from response. Also, in tests is used only string and method `getError()` is used only in tests